src/libstd/lib.rs

```diff
@@ -242,8 +242,8 @@
 #![feature(atomic_mut_ptr)]
 #![feature(box_syntax)]
 #![feature(c_variadic)]
-#![feature(cfg_accessible)]
```

Comment:
cc @Mark-Simulacrum Do we enforce this sort via tidy?

Reply:
We do not, no, and to be honest I think the sorting here is probably overkill, I wouldn't want to enforce it personally. But :shrug: